fmlloader/build.gradle

```diff
@@ -36,6 +37,7 @@ dependencies {
     implementation(libs.terminalconsoleappender)
     implementation(libs.jimfs)
 
+    // Needed because we have a custom log4j plugin, and they removed package scanning and require a data file to be generated
```

Comment:
Unrelated, but could this be part of the reason why 1.18.2 suffered from the Log4J bump? See #10190

Reply:
As far as I know, no because the bump is to a version where this feature is deprecated but functional. Not removed.